tests/test_cases/test_cocotb/test_timing_triggers.py

```diff
@@ -154,21 +157,6 @@ def test_cached_write_in_readonly(dut):
         raise TestFailure
 
 
-@cocotb.test(expect_fail=cocotb.SIM_NAME.lower().startswith(("icarus",
-                                                             "chronologic simulation vcs")),
-             skip=cocotb.SIM_NAME.lower().startswith(("ncsim", "xmsim")))
-def test_afterdelay_in_readonly(dut):
-    """Test doing invalid sim operation"""
-    global exited
-    exited = False
-    clk_gen = cocotb.fork(Clock(dut.clk, 100).start())
-    coro = cocotb.fork(do_test_afterdelay_in_readonly(dut, 0))
-    yield [Join(coro), Timer(1000)]
-    clk_gen.kill()
-    if exited is not True:
-        raise TestFailure
```

Comment:
So, my reading of this test is that it's intended to check that cocotb can recover and keep running the remaining tests if we ask the sim to do something illegal.

I'd therefore argue we should be keeping this test, and we just need to make sure the warning never happens. That would be straightforward to do with
```
with warnings.capture_warnings():
    warnings.simplefilter('ignore')
    t = Timer(0)
...
coro = cocotb.fork(do_test_afterdelay_in_readonly(dut, t))
```

Reply:
I think we do not need that particular Timer(0) scenario for that.

There are other tests that do exactly what you describe:
![grafik](https://user-images.githubusercontent.com/25175069/95597172-96e13300-0a4e-11eb-8070-1420512d3f22.png)

Reply:
That's a good point. Unless @ktbarrett agrees with me, this is fine to leave out.

Reply:
as he originally proposed to take it out and we agreed above I assume that is ok.